packages/react-storage/src/components/StorageManager/ui/FileList/types.ts

```diff
@@ -10,17 +8,11 @@ export interface FileListProps {
   isResumable: boolean;
   onCancelUpload: (params: {
     id: string;
-    uploadTask: Storage.UploadDataOutput;
+    uploadTask: UploadDataOutput;
   }) => void;
   onDeleteUpload: (params: { id: string }) => void;
-  onPause: (params: {
-    id: string;
-    uploadTask: Storage.UploadDataOutput;
-  }) => void;
-  onResume: (params: {
-    id: string;
-    uploadTask: Storage.UploadDataOutput;
-  }) => void;
+  onPause: (params: { id: string; uploadTask: UploadDataOutput }) => void;
+  onResume: (params: { id: string; uploadTask: UploadDataOutput }) => void;
```

Comment:
IMO `uploadTask` name here needs to be reconsidered, and `UploadDataOutput` should be replaced with a scaffold type